src/com/owncloud/android/ui/activity/ShareActivity.java

```diff
@@ -208,7 +208,7 @@ public void refreshUsersOrGroupsListFromServer() {
         // Show loading
         showLoadingDialog(R.string.common_loading);
         // Get Users and Groups
-        GetShareWithUsersAsyncTask getTask = new GetShareWithUsersAsyncTask(this);
+        GetSharesForFileAsyncTask getTask = new GetSharesForFileAsyncTask(this);
```

Comment:
This variable name (getTask) is too generic, you could use getSharesForFileTask instead, or even **getPrivateSharesForFileTask**, which would be consistent with the name of the method getPrivateSharesForAfile.